corehq/apps/registration/views.py

```diff
@@ -183,6 +183,13 @@ def register_new_user(self, data):
                         'project name unavailable': [],
                     }
                 }
+            except ErrorInitializingDomain as e:
+                logging.error(f"Unable to initialize domain during new user signup: {str(e)}")
+                return {
+                    'errors': {
+                        'temporary system issue': [],
```

Comment:
based on the other error format returned, it appears this isn't shown to the user. Also, this would occur pre-login where we currently don't support translation anyway